### ISSUE TITLE
[Bug] UI not displaying properly.

### ISSUE THREAD

**Issue by philippaudier** [BloCamLimb/ModernUI#1]
**Describe the bug**
The UI is not displaying properly. When I press ESC, the new menu is appearing though.

**To Reproduce**
I placed the mod in my mods folder and started the game. I played in a pre-existing world in 1.15.2.

**Expected behavior**
I was expecting to be able to use the new UI. I use OptiForge-MC1.15.2-0.1.8 and play with Chocapik13 V8 ultra shader.

**Screenshots**
The problem:
![2020-04-11_19 16 34](https://user-images.githubusercontent.com/41243470/79050301-62b8c100-7c29-11ea-9d1f-893ac24200c9.png)
F3 infos:
![2020-04-11_19 16 53](https://user-images.githubusercontent.com/41243470/79050305-68160b80-7c29-11ea-904c-ab099652ea1f.png)
Mods installed:
![2020-04-11_19 17 25](https://user-images.githubusercontent.com/41243470/79050307-6ea48300-7c29-11ea-9761-29cbdc8a6873.png)

**Versions you are on**
 - Mod Version: ModernUI-1.15.2-1.3.5.30
 - Forge Version: forge-1.15.2-31.1.39

**Comment by iiKraze** [comment]
you need to change what java version minecraft is running

Add this line into the profile options under **JAVA EXECUTABLES**

C:\Program Files\Java\jre1.8.0_241\bin\javaw.exe

![2020-04-11__11-24-19PM__MinecraftLauncher](https://user-images.githubusercontent.com/46874817/79056191-9f95af80-7c4b-11ea-8a98-b951ca017937.png)

**Comment by iiKraze** [comment]
you need to change what java version minecraft is running

Add this line into the profile options under JAVA EXECUTABLES

C:\Program Files\Java\jre1.8.0_241\bin\javaw.exe
![2020-04-11__11-25-57PM__MinecraftLauncher](https://user-images.githubusercontent.com/46874817/79056207-cd7af400-7c4b-11ea-875a-600421077d73.png)

**Comment by iiKraze** [comment]
you need to change what java version minecraft is running

Add this line into the profile options under JAVA EXECUTABLES

`C:\Program Files\Java\jre1.8.0_241\bin\javaw.exe`

![2020-04-11__11-25-57PM__MinecraftLauncher](https://user-images.githubusercontent.com/46874817/79056215-d8358900-7c4b-11ea-8dee-b8adf90d3624.png)

**Comment by BloCamLimb** [comment]
Yep, like iikraze said, you need to update java version if you're using official launcher

**Comment by philippaudier** [comment]
Hi, thanks for the replies.
So I've updated my Java version to _241 but the game keeps using the 1.8.0_51 version of Java, I don't know why. I've deleted all of the previous Java versions on my computer, but the game is stuck at the u51. And when I specify the Java version to use in my profile, the game crash. And yes, I am using the official launcher.

Screenshots:
My Java version:
![java build 241](https://user-images.githubusercontent.com/41243470/79059089-53ac3000-7c76-11ea-84ae-1f408aba2bc8.PNG)

Path (C:\Program Files (x86)\Java\jre1.8.0_241\bin\javaw.exe):
![javaw](https://user-images.githubusercontent.com/41243470/79059155-1a27f480-7c77-11ea-8fbb-413da4b54743.PNG)
(but the game won't start)

And I've found that the Minecraft launcher has its own javaw.exe:
Minecraft directory:
![mc launcher](https://user-images.githubusercontent.com/41243470/79059241-df728c00-7c77-11ea-94c3-b6906abdb87f.PNG)

Well, at this point, I think I'm on my own now, but if you guys have clues on how to force Minecraft to somehow use my Java version, let me know.
